src/link/linker.1.s

```diff
@@ -496,12 +496,15 @@ getopcode
                 blt       ]lup
                 beq       ]lup
                 dex
-:done           lda       #$2020
+:done           lda       #$2020                    ;truncate to 3 bytes max
                 sta       opcode+$1,Y
+                sta       opcode+4
                 tya
-                and       #$1F
                 sep       $20
-                sta       opcode
+                cmp       #4

```

Comment:
Could we make the limit a preference?

Worth it?

Reply:
This only affects linker script opcode lookup and they will never be more than 3 characters.